src/dali_executor/executor.test.cc

```diff
@@ -29,15 +29,49 @@
 
 namespace triton { namespace backend { namespace dali { namespace test {
 
+template<typename T, typename Op>
+void pointwise_compare(const std::vector<OBufferDescr> &obuffers,
+                       const std::vector<std::vector<T>> &ibuffers, size_t inp_size, const Op &op) {
```

Comment:
This name is not very informative.
```suggestion
void coalesced_compare(const std::vector<OBufferDescr> &obuffers,
                       const std::vector<std::vector<T>> &ibuffers, size_t inp_size, const Op &op) {
```